core/src/translate/query.rs

```diff
@@ -29,10 +29,15 @@ pub fn translate_query(sql_query: &SqlQuery) -> Result<Query> {
     } = sql_query;
 
     let body = translate_set_expr(body)?;
-    let order_by = order_by
-        .iter()
-        .map(translate_order_by_expr)
-        .collect::<Result<_>>()?;
+    let order_by = if let Some(order_by) = order_by {
+        order_by
+            .exprs
+            .iter()
+            .map(translate_order_by_expr)
+            .collect::<Result<_>>()?
+    } else {
+        vec![]
+    };
```

Comment:
nit. it would be good to use flat_map here
```rust
    let order_by = order_by
        .iter()
        .flat_map(|order_by| order_by.exprs.iter().map(translate_order_by_expr))
        .collect::<Result<_>>()?;
```

Reply:
I thought about that but I wasn't sure which one you'd prefer. Okay.

Reply:
Done